virtual_rainforest/models/abiotic/abiotic_tools.py

```diff
@@ -0,0 +1,70 @@
+"""The ``models.abiotic.abiotic_tools`` module contains a set of general functions and
+universal constants that are shared across submodels in the abiotic model.
```

Comment:
Is this deliberately not a link? It might well be, because it would be a link to itself, basically, but just checking.

Reply:
Yes, I followed the syntax from other modules here. I could add a link to the `abiotic_model`, but I think that can wait until I had a think about what else to write in the docstring that is a bit more descriptive. For know it's just to continue with other things more easily.

We should definitely have a general discussion about constants and where they should go, at the moment each abiotic submodule has a data class at the top but this is not meant to stay there, it could well go in an `abiotic.constants` file that constains all data classes.